src/AppInstallerCLITests/CompositeSource.cpp

```diff
@@ -561,3 +557,18 @@ TEST_CASE("CompositeSource_MultipleAvailableSources_ReverseMatchBoth", "[Composi
     REQUIRE(result.Matches[0].Package->GetInstalledVersion());
     REQUIRE(result.Matches[0].Package->GetAvailableVersionKeys().size() == 1);
 }
+
+TEST_CASE("CompositeSource_IsSame", "[CompositeSource]")

```

Comment:
>"CompositeSource_IsSame" [](start = 10, length = 24)

Curious what this test actually do, are we concerned that calling search twice will have different results?

Reply:
This test is to exercise the IsSame function of the CompositeSource packages.